modules/admixerBidAdapter.js

```diff
@@ -1,88 +1,76 @@
-var adaptermanager = require('src/adaptermanager');
+import * as utils from 'src/utils';
+import {registerBidder} from 'src/adapters/bidderFactory';
 
-/**
- * Adapter for requesting bids from Admixer.
- *
- * @returns {{callBids: _callBids,responseCallback: _responseCallback}}
- */
-var AdmixerAdapter = function AdmixerAdapter() {
-  var invUrl = '//inv-nets.admixer.net/prebid.aspx';
-  var invVastUrl = '//inv-nets.admixer.net/videoprebid.aspx';
-
-  function _callBids(data) {
-    var bids = data.bids || [];
-    for (var i = 0, ln = bids.length; i < ln; i++) {
-      var bid = bids[i];
-      var params = {
-        'sizes': utils.parseSizesInput(bid.sizes).join('-'),
-        'zone': bid.params && bid.params.zone,
-        'callback_uid': bid.placementCode
-      };
-      if (params.zone) {
-        if (bid.mediaType === 'video') {
-          var videoParams = {};
-          if (typeof bid.video === 'object') {
-            Object.assign(videoParams, bid.video);
-          }
-          Object.assign(videoParams, params);
-          _requestBid(invVastUrl, params);
-        } else {
-          _requestBid(invUrl, params);
-        }
-      } else {
-        var bidObject = bidfactory.createBid(2);
-        bidObject.bidderCode = 'admixer';
-        bidmanager.addBidResponse(params.callback_uid, bidObject);
+const BIDDER_CODE = 'admixer';
+const ENDPOINT_URL = '//inv-nets.admixer.net/prebid.1.0.aspx';
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: [],
+  supportedMediaTypes: ['banner', 'video'],
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {BidRequest} bid The bid params to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: function (bid) {
+    return !!bid.params.zone;
+  },
+  /**
+   * Make a server request from the list of BidRequests.
+   *
+   * @param {bidderRequest} - bidderRequest.bids[] is an array of AdUnits and bids
+   * @return ServerRequest Info describing the request to the server.
+   */
+  buildRequests: function (bidderRequest) {
+    const payload = {
+      imps: [],
+      referrer: utils.getTopWindowUrl(),
+    };
+    bidderRequest.forEach((bid) => {
+      if (bid.bidder === BIDDER_CODE) {
+        payload.imps.push(bid);
       }
-    }
-  }
-
-  function _requestBid(url, params) {
-    Ajax.ajax(url, _responseCallback, params, {method: 'GET', withCredentials: true});
-  }
-
-  function _responseCallback(adUnit) {
+    });
+    const payloadString = JSON.stringify(payload);
+    return {
+      method: 'GET',
+      url: ENDPOINT_URL,
+      data: `data=${payloadString}`,
+    };
+  },
+  /**
+   * Unpack the response from the server into a list of bids.
+   *
+   * @param {*} serverResponse A successful response from the server.
+   * @return {Bid[]} An array of bids which were nested inside the server.
+   */
+  interpretResponse: function (serverResponse, bidRequest) {
+    const bidResponses = [];
```

Comment:
#1748 changed the first argument of `interpretResponse` to:

```
{
  body: responseBody,
  headers: {
    get: function(header) { /* returns a header from the HTTP response */ }
  }
}
```

so adding something like

`serverResponse = serverResponse.body;`

just below this line, or however you'd prefer to grab the body, and updating corresponding tests if needed should get this back to working properly